### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by ayebaishmo**
Hello microverse set up.
 With in the feature branch I have created html , css files. I have also added Linter files that help me to write proper code.
I have also corrected the linter errors by adding .hintrc and stylelintrc.json which helped me idenify the linter errors and were laiter corrected in the 9th and 10th commit.

### CODE REVIEW

**Comment by Davidosky007** [comment]
Hi @ayebaishmo, 

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I suggest your readme file be more descriptive as stated in the project requirement
* your pull request should have a descriptive title telling what changes are inside and also it could have a short summary describing introduced changes in more details. these make reviewing easy.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 [OPTIONAL]
You can also consider:
- it's best to start your commit messages with capital letters. this is a best practice 
- I suggest the h1 tag be used for the first heading these helps the browser understand the meaning of the content instead of just displaying it. 👍 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


# Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 1]
```diff
@@ -1,2 +1,58 @@
...
+
+# This is git hub project for study prposes about html and css
+
+
+# ![Capture](https://user-images.githubusercontent.com/42030948/120382517-95fc5a80-c2d8-11eb-8928-4a83c036f607.PNG)
+
+
+
+# It is about a web page that is designed using HTML sytanx and css
+Uder Microverse helloe world project
```
Comment (Dipeshtwis) [inline README.md:12]: [Optional]
There are some typo in your README file. Please fix that

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @ayebaishmo, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️

- [ ] your pull request should have a descriptive title telling what changes are inside and also it could have a short summary describing introduced changes in more details. these make reviewing easy. ( Title should be :- **Hello Microverse project setup** and PR description should be about what file you have added, which feature you have introduced and not like what you have written for now). Please fix that.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

_Check the comments under the review._

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @ayebaishmo!

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

doing good

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**